firefox-ios/Client/Frontend/Browser/SearchLoader.swift

```diff
@@ -115,49 +115,59 @@ class SearchLoader: Loader<Cursor<Site>, SearchViewModel>, FeatureFlaggable {
                 }
 
                 DispatchQueue.main.async {
-                    let results = queries
-                    defer {
-                        GleanMetrics.Awesomebar.queryTime.stopAndAccumulate(timerid)
-                    }
+                    self.updateUIWithBookmarksAsSitesResults(queries: queries,
```

Comment:
@ionixjunior seems pretty straight forward as you just moved it to the method below

Reply:
@nbhasin2 yeah, this ensures that the `getBookmarksAsSites` closure remains small and doesn't violate the `closure_body_length` rule.

Reply:
@ionixjunior seems like warnings increased, would you mind rebasing the branch to address it.

Reply:
@nbhasin2 done!